lib/ladybug/energy_model/face.rb

```diff
@@ -99,16 +97,26 @@ def create_openstudio_object(openstudio_model)
             openstudio_surface.setViewFactortoGround(@hash[:boundary_condition][:view_factor])
           end
         when 'Surface'
-          surface = nil 
           if @hash[:boundary_condition][:boundary_condition_objects][0]
+            surface = nil
             surface_object = openstudio_model.getSurfaceByName(@hash[:boundary_condition][:boundary_condition_objects][0])
             unless surface_object.empty?
               surface = surface_object.get
               openstudio_surface.setAdjacentSurface(surface)
```

Comment:
I just realized that, while this original code here won't set the adjacent surface correctly the first time around (ie. when adding surface1 to a Model, which is adjacent to surface2), it will set the adjacency correctly on the second time around (when adding surface2 to the model and setting it to be adjacent to surface1, which should already be in the model by that point). So I think the original code here might actually be ok and I just wasn't thinking correctly about it before. @tanushree04 , in your tests so far, can you see whether the adjacent surfaces of the `model_multi_zone_office.json` model are being set correctly? If so, we can just keep this code the way it is (without the need for the global hash) and you can also get rid of the commented code that you have below here.

Reply:
@chriswmackey you're right, I ran the measure and the surfaces are correctly assigned as adjacent surfaces in the osm using the earlier code.

Reply:
Glad to hear that is the case. Let's just use your original solution, then. I see you commented out the code for the new one we had been considering with the global hash and I would say that we can just delete that code rather than commenting it out (better to keep the code clean). Once you do that, feel free to merge this PR in!